src/main/java/edu/cornell/kfs/concur/batch/businessobject/ConcurStandardAccountingExtractDetailLine.java

```diff
@@ -31,6 +31,10 @@
     private Date reportEndDate;
     private String policy;
     private String expenseType;
+    private KualiDecimal cashAdvanceAmount;
+    private Date cashAdvanceIssueDate;
```

Comment:
OK, I will remove these, I thought I heard in the meeting that you would need them, I guess I misheard